styleguide/backstop_data/ci_report/xunit.xml

```diff
@@ -0,0 +1,2 @@
ion ›› molecules-partner-info (.ama__partner) component" type="CSS component"/></testcase><testcase name=" ›› organisms-partner-list" classname=".ama__partner-list"><error message="Design deviation ›› organisms-partner-list (.ama__partner-list) component" type="CSS component"/><failure message="Design deviation ›› organisms-partner-list (.ama__partner-list) component" type="CSS component"/></testcase><testcase name=" ›› organisms-partner-list" classname=".ama__partner-list"><error message="Design deviation ›› organisms-partner-list (.ama__partner-list) component" type="CSS component"/><failure message="Design deviation ›› organisms-partner-list (.ama__partner-list) component" type="CSS component"/></testcase><testcase name=" ›› organisms-partner-list" classname=".ama__partner-list"><error message="Design deviation ›› organisms-partner-list (.ama__partner-list) component" type="CSS component"/><failure message="Design deviation ›› organisms-partner-list (.ama__partner-list) component" type="CSS component"/></testcase><testcase name=" ›› organisms-partner-list" classname=".ama__partner-list"><error message="Design deviation ›› organisms-partner-list (.ama__partner-list) component" type="CSS component"/><failure message="Design deviation ›› organisms-partner-list (.ama__partner-list) component" type="CSS component"/></testcase><testcase name=" ›› organisms-partner-list" classname=".ama__partner-list"><error message="Design deviation ›› organisms-partner-list (.ama__partner-list) component" type="CSS component"/><failure message="Design deviation ›› organisms-partner-list (.ama__partner-list) component" type="CSS component"/></testcase><testcase name=" ›› topics-page" classname="#sg-viewport"><error message="Design deviation ›› topics-page (#sg-viewport) component" type="CSS component"/><failure message="Design deviation ›› topics-page (#sg-viewport) component" type="CSS component"/></testcase><testcase name=" ›› topics-page" classname="#sg-viewport"><error message="Design deviation ›› topics-page (#sg-viewport) component" type="CSS component"/><failure message="Design deviation ›› topics-page (#sg-viewport) component" type="CSS component"/></testcase><testcase name=" ›› topics-page" classname="#sg-viewport"><error message="Design deviation ›› topics-page (#sg-viewport) component" type="CSS component"/><failure message="Design deviation ›› topics-page (#sg-viewport) component" type="CSS component"/></testcase><testcase name=" ›› topics-page" classname="#sg-viewport"><error message="Design deviation ›› topics-page (#sg-viewport) component" type="CSS component"/><failure message="Design deviation ›› topics-page (#sg-viewport) component" type="CSS component"/></testcase><testcase name=" ›› topics-page" classname="#sg-viewport"><error message="Design deviation ›› topics-page (#sg-viewport) component" type="CSS component"/><failure message="Design deviation ›› topics-page (#sg-viewport) component" type="CSS component"/></testcase></testsuite></testsuites>
```

Comment:
Please remove this from git with `git rm -r --cached styleguide/backstop_data/ci_report/` so that it's removed from the git history.